pkg/metrics/metrics.go

```diff
@@ -180,6 +194,18 @@ var (
 		Help:      "Total forwarded packets, tagged by ingress/egress direction",
 	},
 		[]string{"direction"})
+
+	// Datapath statistics
+
+	// DatapathManagementErrors is the number of errors managing BPF maps
+	// and so on in the last period.
+	DatapathManagementErrors = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Despite the already existing metrics, I think gauges aren't expected to have _count on the end. This can be a counter, however (with _total on the end) since it's guaranteed to increase (0 rollovers on restart are handled, I think).
See https://prometheus.io/docs/practices/naming/ for some more fun. @manalibhutiyani recently went through this so she is also a good resource.

Reply:
+1